Packs/ExpanseV2/TestPlaybooks/playbook-ExpanseV2_Test.yml

```diff
@@ -3570,12 +3397,13 @@ inputs:
     simple: 12.2.248.97
   required: false
   description: 'IP address '
-  playbookInputQuery: null
+  playbookInputQuery:
 - key: common_name
   value:
     simple: api-dev.radioshack.com
   required: false
   description: A partial certificate common name
-  playbookInputQuery: null
+  playbookInputQuery:
 outputs: []
 fromversion: 6.0.0
+version: -1
```

Comment:
Please run demisto-sdk format on this TPB.

Reply:
It's been done